contract_manager/src/contracts/evm.ts

```diff
@@ -321,6 +364,43 @@ export class EvmEntropyContract extends Storable {
     return new EvmEntropyContract(chain, parsed.address);
   }
 
+  //TODO: document and move this to executor contract
```

Comment:
I think it's ok to keep these in this contract but please add some documentation and also merge the logic of these 2 functions as much as possible. All governance stuff will have the same `new EvmExecute` boilerplate code which can be shared.